docker/service/src/main/java/uk/nhs/adaptors/scr/utils/DateUtil.java

```diff
@@ -0,0 +1,34 @@
+package uk.nhs.adaptors.scr.utils;
+
+import static uk.nhs.adaptors.scr.utils.ThrowFhirMappingError.throwFhirMappingError;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.nhs.adaptors.scr.exceptions.FhirMappingException;
+
+@Slf4j
+public class DateUtil {
+    public static String formatDate(String dateStr) throws FhirMappingException {
+        String value = "";
+
+        if (dateStr != null) {
+            if (!dateStr.isBlank()) {
+                SimpleDateFormat sdfInput = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
```

Comment:
both date `SimpleDateFormat`s can be consts or extract patterns as consts

Reply:
resolved